### DISCUSSION TITLE
Will Wordpress sites be supported?

### DISCUSSION THREAD

**Discussion by WesleyLiaw777** [lncrawl/lightnovel-crawler#1895] [Ideas]
I tried to get A Practical Guide to Evil into an epub, but didn't have any success. Is it possible for the lncrawl code to be adapted to manage this kind of website layout? I've included the webpage below.
https://practicalguidetoevil.wordpress.com/table-of-contents/ 
Thanks in advance!

**Comment by dipu-bd** [comment]
If all of the wordpress sites would be using the same layout, it would be possible. We try to indentify similar sites and create a common layout code for them. If you have a group of links that uses the same layouts, you can share them. Without at least 3-4 similar sites, creating a layout is not worth it.
